privacyidea/api/before_after.py

```diff
@@ -266,7 +266,21 @@ def after_request(response):
 @postrequest(sign_response, request=request)
 def auth_error(error):
     if "audit_object" in g:
-        g.audit_object.log({"info": error.message})
+        message = ''
+        description = ''
+        
+        if hasattr(error, 'message'):
+            message = error.message
+            
+        if hasattr(error, 'description'):
```

Comment:
Hmm, if I understand correctly ``auth_error`` is only called for errors that are instances of ``AuthError``. But it seems that ``AuthError`` have no ``description`` or ``details`` attributes, only ``message``. Wouldn't this mean that the first ``if`` branch would be enough?

Reply:
Sorry @p53, I was mistaken, ``AuthError`` does have a ``details`` attribute. :-) But (unless I'm mistaken again) it does not have a ``description`` attribute?

Reply:
This code is a bit misleading -- if you look at ``privacyIDEAError.__init__``, you see that the ``description`` argument actually gets assigned to the ``message`` instance variable. Thus, a ``privacyIDEAError`` instance actually has no ``description``. So I guess you could remove this ``if`` branch. This will also increase coverage (because right now, the branch is not hit by any unit tests).

Reply:
ah, i didn't expect that, didn't check privacyIDEAError